### PR TITLE
elm-format 0.2.0 (new formula)

### PR DESCRIPTION
**Pull request by enilsen16**

### CODE REVIEW

File: Formula/elm-format.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+class ElmFormat < Formula
+  include Language::Haskell::Cabal
+  desc "Elm source code formatter, inspired by gofmt."
+  homepage "https://github.com/avh4/elm-format"
+
+  head "https://github.com/avh4/elm-format.git"
+
+  devel do
+    url "https://github.com/avh4/elm-format/archive/0.2.0-alpha.tar.gz"
+    version "3"
```
Comment (bfontaine) [inline Formula/elm-format.rb]: We can keep upstream’s version numbering; so that’s version `0.2.0-alpha`, not `3`.

Comment (enilsen16) [inline Formula/elm-format.rb]: :+1:

**Comment by fobos** [comment]
Vote for this!

**Comment by enilsen16** [comment]
@bfontaine Looks like the build is failing because this is a devel-only formula...

**Comment by bfontaine** [comment]
That was expected; thank you @enilsen16! :tada:

**Comment by kylecordes** [comment]
By the way, I stumbled across this recently, and found it was quite helpful in a first appeared but now it is a red herring, long behind the development of elm-format itself. New line

**Pull request closed without merging**